source/content/guides/secrets/02-basic-concepts.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Pantheon Secrets Guide
+subtitle: Basic Concepts
+description: Gaining familiarity with some concepts about Pantheon Secrets will help you make the most of this feature.
+contributors: [stovak]
+contenttype: [guide]
+innav: [true]
+categories: [secrets]
+cms: [drupal, wordpress]
+audience: [development]
+product: [secrets]
+integration: [--]
+tags: [reference, cli, local, terminus, workflow]
+permalink: docs/guides/secrets/basic-concepts
+reviewed: "2024-05-01"
+---
+<dl>
+
+<dt>Secret</dt>
+
+<dd>
+
+A key-value pair that should not be exposed to the general public, typically something like a password, API key, or other sensitive information that should not be added to version control.
+
+</dd>
+
+<dt>Secret type<sup>1</sup></dt>
+
+<dd>
+This represents how the secret is used.  A secret can only have one type.
+
+Current types are:
+
+  * `runtime`: This secret type can be retreived directly from your application code.  This is the recommended type if you want your application to be able to use the secret while it's operating.
+
+  * `env`: This type is used to set environment variables. Environment variables are currently only supported for Integrated Composer builds; setting environment variables on the application server is unsupported.
```

Comment:
Once LOPS-1977 is done, we'll need to update this to describe how users can opt-in to use secrets of type "env" from the "web" scope. Point out that secrets of type env should not be considered to be encrypted at rest, and that  environment variables are easy (easier?) to accidentally expose.